build.gradle.kts

```diff
@@ -10,16 +10,15 @@ plugins {
 buildscript {
     repositories {
         google()
-        jcenter()
```

Comment:
Removing jcenter will not work properly because `koin` v2 is only available on jcenter.

Reply:
Koin have not been `classpath`ed in `buildscript`'s dependencies, subproejcts `implementation` it with jcenter.

Reply:
And I just moved `subprojects` repositories into `allprojects` closure

![image](https://user-images.githubusercontent.com/10363352/113704232-739df480-970e-11eb-8a98-74274272b692.png)